sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -100,7 +100,7 @@
  * An unbounded source and a sink for <a href="http://kafka.apache.org/">Kafka</a> topics.
  * Kafka version 0.9 and 0.10 are supported. If you need a specific version of Kafka
  * client(e.g. 0.9 for 0.9 servers, or 0.10 for security features), specify explicit
- * kafka-client dependency.
+ * kafka-client dependency with Maven property ${kafka.clients.version}.
```

Comment:
I think we should remove this update. Uses are not building beam or beam modules by themselves, rather are just using published mvn artifacts. This property is not irrelevant to them.